pipelines/legionPipeline.groovy

```diff
@@ -187,10 +193,13 @@ def setupGcpAccess() {
     sh """
         set -ex
         # Authorize GCP access
-        gcloud auth activate-service-account --key-file=${gcpCredential} --project=${env.param_gcp_project}
+        gcloud auth activate-service-account --key-file=${gcpCredential} --project=${gcp_project_id}
 
         # Setup Kube api access
-        gcloud container clusters get-credentials ${env.param_cluster_name} --zone ${env.param_gcp_zone} --project=${env.param_gcp_project}
+        gcloud container clusters get-credentials ${env.param_cluster_name} --zone ${gcp_zone}
+
+        # Show k8s context
+        kubectl config get-contexts |grep ${env.param_cluster_name} || echo "Warning: No cluster context found!"
```

Comment:
Do we need this command?

Reply:
thought that may help for NoneType issue debug, I'll remove it since the problem is not in missed context.